cdhweb/people/migrations/0011_populate_nonproxy_person.py

```diff
@@ -28,15 +28,22 @@ def create_nonproxy_persons(apps, schema_editor):
                 'institution': profile.institution,
                 'pu_status': user.profile.pu_status
             })
-        Person.objects.create(**person_info)
-
-    # NOTE: prune users who don't have login access? later?
-
-    # TODO: do we care about log entries for person creation?
-    # get script user and log entry model to document changes
-    # script_user = User.objects.get(username=settings.SCRIPT_USERNAME)
-    # LogEntry = apps.get_model('admin', 'LogEntry')
-    # ContentType = apps.get_model('contenttypes', 'ContentType')
+        person = Person.objects.create(**person_info)
+
+        # log that the person was created via this migration script
+        LogEntry = apps.get_model('admin', 'LogEntry')
+        ContentType = apps.get_model('contenttypes', 'ContentType')
+        script_user = User.objects.get(username=settings.SCRIPT_USERNAME)
+        person_contenttype = ContentType.objects.get(
+            app_label='people', model='person')
```

Comment:
These lines should be moved outside the for loop, but otherwise this looks good.

Reply:
done